app/models/work.rb

```diff
@@ -44,8 +50,9 @@ class Work < ApplicationRecord
       transition pending_approval: :first_draft
     end
 
-    event :new_version do
+    event :update_metadata do
```

Comment:
Could this event also include changes to the uploaded files? I'm wondering if `new_version` is more or less descriptive than `update_metadata` here.

Reply:
I think new_version is not the right event, because the event is "the user presses the save button"

Reply:
This event is fired whenever the user clicks the "Deposit" button after the initial press of the "Deposit" button, right? So it's not just metadata. But to the end user, it's not different from the first press of the "Deposit" button.

Reply:
I'm OK with moving away from the event service, but I think I'd benefit from pairing a bit so we can discuss design.